inference/core/env.py

```diff
@@ -236,6 +239,12 @@
 # Metlo key, default is None
 METLO_KEY = os.getenv("METLO_KEY", None)
 
+DEVICE_MANAGER_USERNAME = os.getenv("DEVICE_MANAGER_USERNAME")
+DEVICE_MANAGER_PASSWORD = os.getenv("DEVICE_MANAGER_PASSWORD")
```

Comment:
Used to authenticate with the pubsub host. In the next iteration, this is going to be mapped to the API_KEY, so user's won't need to explicitly add it